tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2089,7 +2109,14 @@ public function userRemovesAllSharesFromTheFileNamedzz($user, $fileName) {
 	public function userHasRemovedAllSharesFromTheFileNamedzz($user, $fileName) {
```

Comment:
```suggestion
	public function userHasRemovedAllSharesFromTheFileNamed($user, $fileName) {
```
While you are in this function, please check why it might be named with `zzz` at the end. If there is no good reason, then remove the `zz`

Reply:
fixed